packages/create-app/index.js

```diff
@@ -2,10 +2,40 @@
 /* eslint-disable node/shebang */
 
 const { yellow, green } = require('kolorist')
+
+const alternativeCommands = {
+  yarn: 'yarn create vite',
+  pnpm: 'pnpx create-vite-app',
```

Comment:
Isn't it just `pnpx create-vite`?
Otherwise we need to change the docs https://vitejs.dev/guide/#scaffolding-your-first-vite-project
_I'm not a pnpm user ... So I don't know myself_

Reply:
@Shinigami92 
Sorry.
This is my mistake.